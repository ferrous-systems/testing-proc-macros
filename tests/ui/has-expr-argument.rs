use contracts::contracts;

#[contracts(true)]
fn f() {}

fn main() {}
