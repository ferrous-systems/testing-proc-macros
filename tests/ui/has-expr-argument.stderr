error: this attribute takes no arguments

         = help: use `#[contracts]`

 --> $DIR/has-expr-argument.rs:3:13
  |
3 | #[contracts(true)]
  |             ^^^^
